examples/api/lib/widgets/magnifier/cupertino_text_magnifier.0.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+
+void main() => runApp(const TextMagnifierExampleApp(text: 'Hello world!'));
+
+class TextMagnifierExampleApp extends StatefulWidget {
+  const TextMagnifierExampleApp({
+    super.key,
+    required this.text,
+  });
+
+
+  final String text;
+
+  @override
+  State<TextMagnifierExampleApp> createState() => _TextMagnifierExampleAppState();
+}
+
+class _TextMagnifierExampleAppState extends State<TextMagnifierExampleApp> {
+  final MagnifierController _controller = MagnifierController();
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoApp(
+      theme: const CupertinoThemeData(brightness: Brightness.light),
+      home: CupertinoPageScaffold(
+        navigationBar: const CupertinoNavigationBar(
+        middle: Text('CupertinoTextMagnifier Sample'),
+      ),
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 48.0),
```

Comment:
There are some format issues.

Reply:
Hi, TahaTesser! 
So, can I fix those issues just by using the IDE's "format document", isn't it?